src/extensions/FullscreenExtension/FullscreenExtension.ts

```diff
@@ -16,12 +16,12 @@ export class FullscreenExtension extends Module {
 
   constructor(instance: IInstance) {
     super(instance);
-
-    if (screenfull.enabled) {
+    
+    if (screenfull && screenfull.enabled) {
```

Comment:
Is there a scenario where screenfull is undefined / null? It comes directly from the screenfull import defined on top.